tests/testthat/test-shape-translate.r

```diff
@@ -0,0 +1,81 @@
+context("translate_shape_string")
+
+test_that("strings translate to their corresponding integers", {
+  shape_strings <- c(
```

Comment:
There's no need to systematically test ever possible value - the main thing you want to achieve in this test is to make sure everything is plumbed together correctly.